blip-tap.md

```diff
@@ -1229,7 +1229,10 @@ where:
   fractional representation.
 
 When serialized into a wire message, the fixed-point number is represented as a
-concatenation of its coefficient and scale fields: `<uint64><uint8>`.
+concatenation of its scale and coefficient fields: a 1-byte unsigned integer
+for the scale, followed by a variable-length byte sequence that encodes the
+coefficient as a big-endian unsigned integer. The maximum coefficient size is
```

Comment:
This should be reflected in the wire message definition. 

So we can define a `FixedPoint` serialization, then reference that in the various tables.

Reply:
In the latest commit I've added a section `Fixed-Point Number Serialization` and I've linked to it from the request/accept TLV tables `FixedPoint` field type. I've also used `FixedPoint` in the section body so ctrl+f leads to the relevant serialization section also.

Please let me know if you think this needs more.